packages/cli/README.md

```diff
@@ -46,28 +46,15 @@ will match the input format unless otherwise specified.
 | input | i | Specify a filename for input. |
 | output | o | Specify a filename for output. The extension of this file will control its type unless overridden with `--json` or `--yaml`. |
 
-## Configuration
-
-The CLI can be configured by creating a YAML file called `config.yaml` in the
-following location:
-
-The contents of the config.yaml file should appear as follows
-
-```json
-default:
-	token: xxxx-xxx-xxxx
-```
-
-* `$HOME/.config/@smartthings/cli` on OS/X or Linux
-* `%LOCALAPPDATA%\@smartthings\cli` on Windows
-
 ## Authentication
 
 The CLI supports an automatic login flow that pops up a browser window
 asking you to log in and give the CLI permission to access your account.
 The CLI will automatically request login as needed.
 
-More details about configuration of the CLI can be found on the [configuration documentation page](doc/configuration.md).
+Although not recommended, you can use a personal access token for authentication
+by creating a  [configuration file](doc/configuration.md) and including the
```

Comment:
There is a double space before the link.

Reply:
fixed.